docs/kustomization.yaml

```diff
@@ -87,6 +87,9 @@ secretGenerator:
     username: "curl -s https://path/to/secrets/username.yaml"
     password: "curl -s https://path/to/secrets/password.yaml"
   type: Opaque
+- name: env_file_secret
+  envCommand: printf \"DB_USERNAME=admin\nDB_PASSWORD=somepw\"
```

Comment:
Can you also add some description for `envCommand`? What does it do and how one can use it?
Unlike commands, where it contains multiple key-value pairs, you can only specify one `envCommand`.